.github/workflows/circuits_test.yaml

```diff
@@ -4,7 +4,7 @@ on: [push]
 
 jobs:
   test:
-    name: nargo test
```

Comment:
It feels like this is better suited for static checks workflow as it already does all of our static checks like linting & formatting